packages/react-i18n/src/index.jsx

```diff
@@ -0,0 +1,142 @@
+/**
```

Comment:
File ext should be just `.js` I think.

Reply:
Even if the file contains JSX markup? Apparently it's Gutenberg convention to always use `*.js`. I'll fix that.